crates/wasmparser/src/validator/component.rs

```diff
@@ -2475,6 +2485,15 @@ impl ComponentState {
                         }
                     };
                 }
+                CanonicalOption::Gc => {
```

Comment:
Can this also check for `self.features.cm_gc()` being enabled?

Reply:
Ah yes indeed, thanks